gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageTest.java

```diff
@@ -2826,4 +2852,54 @@ public int read() throws IOException {
       }
     };
   }
+
+  /** Fake AuthorizationHandler to test instantiation behavior. */
+  public static class TestConfAuthorizationHandler implements AuthorizationHandler {
+    public static String PROPERTY_KEY = "required.property";
+    public static String EXPECTED_VALUE = "required-value";
+
+    @Override
+    public void setConf(Map<String, String> property) {
```

Comment:
Rename this method to `setProperties` to be consistent with GCS connector config. Also `setConf` usually assumes that parameter will be Hadoop `Configuration` object, but this is not the case.